lib/JMAP/TestSuite/JMAP/Tester/Wrapper.pm

```diff
@@ -1,168 +1,12 @@
 package JMAP::TestSuite::JMAP::Tester::Wrapper;
```

Comment:
Let's give this a new name like Tester::WithSuiteSuiteSugar.  Wrapper sounds like just a wrapper.